src/audio/Kconfig

```diff
@@ -561,6 +561,12 @@ config PCM_CONVERTER_FORMAT_S24LE
 	help
```

Comment:
git commit message still mentions "S24_4BE"

Reply:
thanks, I will refine it